scripts/release.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -xe
+
+RELEASE="$(git describe --tags)"
+
+go get -u -v ./... || true
```

Comment:
Why the `|| true`?

Reply:
"go get" sometime returns error return code even when no real error, and because of the "set -xe", fails all the script :cry: